packages/terra-slider/src/Slider.module.scss

```diff
@@ -0,0 +1,212 @@
+@import '~terra-mixins/lib/Mixins';
+
+// Themes
+@import './orion-fusion-theme/Slider.module';
+@import './clinical-lowlight-theme/Slider.module';
+
+:local {
+  .slider {
+    align-items: center;
+    display: inline-flex;
+    min-height: 44px;
+  }
+
+  .label {
+    color: var(--terra-slider-label-color, #1c1f21);
+    font-weight: 700;
+    margin-right: 7px;
+    text-align: center;
+    white-space: initial;
+    width: 20%;
```

Comment:
Why is the width 20%.  Wouldn't a label just take up the space necessary?  This seems like an i18n issue.

Reply:
Mentioned in callouts. The label will wrap to the next line in case of long label text. Like here:
https://github.com/cerner/terra-framework/pull/1822/files#diff-20e91930c1eda6dd71cdd1e5ab6f0cf0631a00ffe4ea4c2dc02f2a9c06eaaf0d

![image](https://github.com/cerner/terra-framework/assets/19804033/4ad6aea9-905a-473d-98bb-57f42db84cb8)

Reply:
You don't have the percentages set for the other components though.